Lib/asyncio/timeouts.py

```diff
@@ -52,10 +52,10 @@ def reschedule(self, when: Optional[float]) -> None:
             self._timeout_handler = None
         else:
             loop = events.get_running_loop()
-            self._timeout_handler = loop.call_at(
-                when,
-                self._on_timeout,
-            )
+            if loop.time() >= when:
```

Comment:
I don't know if it's just me, but I find it reads better if we reverse the condition, like so:
```suggestion
            if when <= loop.time():
```